flexmeasures/data/models/task_runs.py

```diff
@@ -23,3 +23,19 @@ def __repr__(self):
             self.datetime,
             {True: "ok", False: "err"}[self.status],
         )
+
+    @staticmethod
+    def record_run(task_name: str, status: bool):
+        """
+        Record the latest task run (overwriting previous ones).
+        If the row is not yet in the table, create it first.
+        Does not commit.
```

Comment:
I guess you mean one should use this in a transaction to have the record committed?

Reply:
That's what people probably would do. Someone has to call `commit` but this function isn't doing it.